Civi/Api4/Generic/Traits/SavedSearchInspectorTrait.php

```diff
@@ -24,6 +24,12 @@ trait SavedSearchInspectorTrait {
    */
   protected $savedSearch;
 
+  /**
+   * An array containing the searchDisplay definition
+   * @var array
+   */
+  protected $display;
```

Comment:
Please don't add this here; it's not relevant to every API that uses this trait.

Reply:
Yes, confirmed the variable should not be in the trait. That helper function is in the trait for convenience but isn't needed by every api action that uses the trait - those that don't use the variable also don't use that function.

Reply:
Thanks - updated